api/src/stampy_chat/logging.py

```diff
@@ -0,0 +1,29 @@
+import json
+from logging import *
+from stampy_chat.env import LOG_LEVEL
+
+
+class ChatLogger(Logger):
+
+    def is_debug(self):
+        return self.isEnabledFor(DEBUG)
+
+    def query(self, query):
+        self.info('query: %s', query)
+
+    def response(self, response):
+        self.info('response: %s', response)
+
+    def moderation_issue(self, query, prompt_string, mod_res):
```

Comment:
This function is incredible single purpose, I would recommend inlining into `check_openai_moderation` to simplify logic and readability.